.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+exclude = .git
```

Comment:
Great!

Also please add this line, `import-order-style=pycharm`, this will set ordering of the modules.

Reply:
Line added as in flask-phbb3.